src/site/content/en/blog/intersectionobserver-v2/index.md

```diff
@@ -0,0 +1,240 @@
+and ☑️ activate "trick mode").
+You will notice that the ad in the iframe "thinks" it received legitimate clicks, even if it was
+completely transparent when you (pretendedly involuntarily) clicked it.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/oSQhudX6lenD4Pld7PJD.png", alt="Tricking a user into clicking an ad by styling it transparent and overlaying it on top of something attractive.", width="800", height="533" %}
+
+## How does Intersection Observer&nbsp;v2 fix this?
+
+Intersection Observer&nbsp;v2 introduces the concept of tracking the actual "visibility" of a target
+element as a human being would define it.
+By setting an option in the
+[`IntersectionObserver` constructor](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/IntersectionObserver),
+intersecting
+[`IntersectionObserverEntry`](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserverEntry)s
+(pardon the wrong plural ending here) will then contain a new boolean field named `isVisible`.
+A `true` value for `isVisible` is a strong guarantee from the underlying implementation
+that the target element is completely unoccluded by other content
+and has no visual effects applied that would alter or distort its display on screen.
+In contrast, a `false` value means that the implementation cannot make that guarantee.
+
+An important detail of the
+[spec](https://w3c.github.io/IntersectionObserver/v2/#calculate-visibility-algo)
+is that the implementation *is permitted* to report *false negatives* (that is, setting `isVisible`
+to `false` even when the target element is completely visible and unmodified).
+For performance or other reasons, implementations should limit themselves to working with bounding
+boxes and rectilinear geometry; they shouldn't try to achieve pixel-perfect results for
+modifications like `border-radius`.
+
+That said, *false positives* are *not permitted* under any circumstances (that is, setting
+`isVisible` to `true` when the target element is not completely visible and unmodified).
+
+{% Aside 'warning' %}
+  Visibility is *much more expensive* to compute than intersection. For that reason,
+  Intersection Observer&nbsp;v2 is *not intended to be used broadly* in the way that
+  Intersection Observer&nbsp;v1 is. Intersection Observer&nbsp;v2 is focused on combatting fraud
+  and should be used only when Intersection Observer&nbsp;v1 functionality is *truly* insufficient.
+{% endAside %}
+
+## What does the new code look like in practice?
+
+The `IntersectionObserver` constructor now takes two additional configuration properties: `delay`
+and `trackVisibility`.
+The `delay` is a number indicating the minimum delay in milliseconds between notifications from
+the observer for a given target.
+The `trackVisibility` is a boolean indicating whether the observer will track changes in a target's
+visibility.
+
+⚠️ It's important to note here that when `trackVisibility` is `true`, `delay` is required to be at
```

Comment:
At the very least, remove this symbol. Possibly turn this paragraph into a note.

Reply:
Emoji be gone.